charts/deepgram-self-hosted/README.md

```diff
@@ -92,6 +92,60 @@ To configure a specific storage option, see the `engine.modelManager.volumes` [c
 
 For detailed instructions on setting up and configuring each storage option, refer to the [Deepgram self-hosted guides](https://developers.deepgram.com/docs/kubernetes) and the respective cloud provider's documentation.
 
+### Service Configuration
```

Comment:
This is great documentation, thanks for writing it up!

We auto-generate the README for this repo to pull in source material from a few different files.

Could you move this human-written content into `README.md.gotmpl`, and then run `helm-docs`? That will auto-generate the README, pulling in this content, as well as documentation for all the fields you added.